test/mobile/features/ios/ios_sessions.feature

```diff
@@ -0,0 +1,55 @@
+Feature: iOS smoke tests for sessions
+
+    # TODO: Failing steps commented out pending full implementation of PLAT-6372
```

Comment:
This looks like it might be obsolete

Reply:
Ah yes.

Reply:
.... removed on my next branch to avoid another CI run.